ijwb/ijwb.bazelproject

```diff
@@ -14,7 +14,7 @@ targets:
 workspace_type: intellij_plugin
 
 build_flags:
-  --define=ij_product=intellij-2024.3
+  --define=ij_product=intellij-ue-2025.1
```

Comment:
it would be nice to see this pointing to `oss-under-dev` as well like you've done for CLion

Reply:
sorry missed this, please refer to #7237